### DISCUSSION TITLE
why external scripts cannot be loaded?

### DISCUSSION THREAD

**Discussion by jojo8356** [brython-dev/brython#2230] [Q&A]
why external scripts cannot be loaded?

**Comment by PierreQuentel** [comment]
They _can_ be loaded by setting the attribute `src` of the tag `<script type="text/python">`. This is explained in the [documentation](https://brython.info/static_doc/en/intro.html).

If it doesn't work for you, can you give more information ?

Beware of the limitation: the loaded script must be in the same domain as the HTML page (this is because it is loaded by an Ajax call).

**Reply by jojo8356** [reply]
give me a example please

**Reply by PierreQuentel** [reply]
It is asked with such grace that I could not refuse

```xml
<!doctype html>
<html>
<head>
<meta charset="utf-8">
<script type="text/javascript" src="/src/brython.js"></script>
</head>

<body onLoad="brython()">

<script type="text/python" src="a.py"></script>

</body>
</html>
```

**Reply by jojo8356** [reply]
and a example brython please "a.py"
